csrankings-d.csv

```diff
@@ -1469,7 +1469,7 @@ Duong Hieu Phan,Ecole Normale Superieure,http://www.di.ens.fr/users/phan,8n2aNu8
 Durga Toshniwal,IIT Roorkee,http://www.iitr.ac.in/departments/CSE/pages/People+Faculty+Durga_Toshniwal.html,NOSCHOLARPAGE
 Dusit Niyato,Nanyang Technological University,http://www.ntu.edu.sg/home/dniyato,T8sVhLMAAAAJ
 Dusit Tao Niyato,Nanyang Technological University,http://www.ntu.edu.sg/home/dniyato,T8sVhLMAAAAJ
-Dustin Richmond,Univ. of California - Santa Cruz,https://www.dustinrichmond.com/,U9hK8g4AAAAJ
+Dustin Richmond,Univ. of California - Santa Cruz,https://www.dustinrichmond.com/,U9hK8g4AAAAJ

```

Comment:
Nothing changed here. Not sure why it shows up in the diff ...

Reply:
Don't worry about it.